### PR TITLE
Read hash data from file instead of DB

### PR DESCRIPTION
**Pull request by alavrovinfb**
- [x] refactored integrity check
- [x] added file storage
- [x] used minio as objects storage
- [x] clean up DB related code

### CODE REVIEW

**Comment by alavrovinfb** [comment]
How to enable Minio server look at README
create snapshot 
rename snapshot file to   <process image>.<algorithm> e.g. nginx:latest.sha256
put snapshot file to minio bucket `integtrity`

File: helm-charts/app-to-monitor/templates/deployment.yaml [hunk 1]
```diff
@@ -61,7 +61,8 @@ spec:
             - name: DEPLOYMENT_TYPE
               value: deployment
           args:
-            - --monitoring-options={{ .Values.configMap.processName }}={{ join "," .Values.configMap.mountPaths }}
+            - --monitoring-options={{ .Values.configMap.processName }}={{ join "," .Values.configMap.monitoringPaths }}
```
Comment (siarhei-shliayonkin) [inline helm-charts/app-to-monitor/templates/deployment.yaml]: minor notice: Join.. used twice

File: internal/data/file_storage.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
+package data
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"strings"
+)
+
+type FileStorage struct {
```
Comment (siarhei-shliayonkin) [inline internal/data/file_storage.go]: With current implementation it looks like reader, not storage. Maybe rename this?

Or will it be extended later?

File: pkg/minio/minio.go [hunk 3]
```diff
@@ -42,6 +42,8 @@ func init() {
 	}
 }
 
+var MS *Storage
```
Comment (siarhei-shliayonkin) [inline pkg/minio/minio.go]: Warn: Looks like uninitialized!

By the way, this var is unnecessary as another var `instance` and the function `Instance()` have the same functionality. You may use them in your code as `minio.Instance()`

File: pkg/minio/minio.go [hunk 4]
```diff
@@ -165,3 +167,7 @@ func (s *Storage) CreateBucketIfNotExists(ctx context.Context, bucketName string
 func (s *Storage) ListBuckets(ctx context.Context) ([]minio.BucketInfo, error) {
 	return s.client.ListBuckets(ctx)
 }
+
+func GetMinioStorage() *Storage {
```
Comment (siarhei-shliayonkin) [inline pkg/minio/minio.go]: Unnecessary. See the previous comment.

File: pkg/minio/minio.go (hunk 3 above)
Comment (alavrovinfb) [reply pkg/minio/minio.go]: thx for advice, I'll take a look

File: helm-charts/app-to-monitor/templates/deployment.yaml (hunk 1 above)
Comment (alavrovinfb) [reply helm-charts/app-to-monitor/templates/deployment.yaml]: done

File: internal/data/file_storage.go (hunk 2 above)
Comment (alavrovinfb) [reply internal/data/file_storage.go]: renamed

File: pkg/minio/minio.go (hunk 3 above)
Comment (alavrovinfb) [reply pkg/minio/minio.go]: done

File: pkg/minio/minio.go (hunk 4 above)
Comment (alavrovinfb) [reply pkg/minio/minio.go]: done

**Comment by alavrovinfb** [comment]
ready for review

**Review by alexwatcher — APPROVED** [review]

**Pull request merged**